dtfs-central-api/src/services/repositories/utilisation-reports-repo.js

```diff
@@ -39,26 +39,25 @@
 
   const utilisationReportDetailsCollection = await db.getCollection(DB_COLLECTIONS.UTILISATION_REPORTS);
   const savedDetails = await utilisationReportDetailsCollection.insertOne(utilisationReportInfo);
-  return { reportId: savedDetails?.insertedId?.toString(), dateUploaded: utilisationReportInfo.dateUploaded };
+  return { reportId: savedDetails.insertedId.toString(), dateUploaded: utilisationReportInfo.dateUploaded };
 };
 
 /**
- * Saves the utilisation report details but not data to the database.
- * @param {string} bankId - Id of the bank.
- * @param {number} month - Month of utilisation report.
- * @param {number} year - Year of utilisation report.
- * @returns {Object | null} - Utilisation report details matching the bank/month/year combo or null if it doesn't exist.
+ * Gets a single utilisation report (not data) by bank ID, month (1-indexed) and year
+ * @param {string} bankId - ID of the bank.
+ * @param {number} month - Month of utilisation report reporting period
+ * @param {number} year - Year of utilisation report reporting period
+ * @returns {Promise<UtilisationReport | null>} - Utilisation report details matching the bank/month/year combo or null if it doesn't exist.
  */
-const getUtilisationReportDetailsForMonthAndYear = async (bankId, month, year) => {
+const getUtilisationReportDetailsByBankIdMonthAndYear = async (bankId, month, year) => {
   const utilisationReportDetailsCollection = await db.getCollection(DB_COLLECTIONS.UTILISATION_REPORTS);
-  const matchingReportDetails = await utilisationReportDetailsCollection.findOne({ 'bank.id': bankId, month, year });
-  return matchingReportDetails;
+  return await utilisationReportDetailsCollection.findOne({ 'bank.id': bankId, month, year });
```

Comment:
I'm not exactly sure how mongo with handle anything dodgy, but we should be validating these values if they are provided in a request (e.g. by the route express validators or otherwise). For the new use case where I've used this the values are generated internally (see "getReportPeriodStartForSubmissionMonth" method) so that's not an issue.